tcf_website/tests/test_review.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=no-member
+"""Tests for Review model."""
```

Comment:
Running the tests locally gives me this:

![image](https://user-images.githubusercontent.com/55100084/105947047-6392d600-6036-11eb-8e62-85946e4ba801.png)

It looks like tests are passing, but as you can see there's quite a bit of junk that's also printed that I don't think we want to be there...